recipes/r-doubletfinder/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: r-doubletfinder
+  version: 1.0.0
+
+source:
+  url: https://github.com/chris-mcginnis-ucsf/DoubletFinder/archive/03e9f37f891ef76a23cc55ea69f940c536ae8f9f.zip
+  sha256: 431b1199f8f18dfb9c2d5335cb174615ce580e4095324b761a0a2dd278c2447b
+
+build:
+  number: 0
+  noarch: generic
+  run_exports: '{{ pin_compatible("r-doubletfinder", max_pin="x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+
+requirements:
+  host:
+    - r-base
+    - r-seurat
+    - r-fields
+    - r-kernsmooth
+    - r-rocr
+  run:
+    - r-base
+    - r-seurat
+    - r-fields
+    - r-kernsmooth
+    - r-rocr
+
+test:
+  commands:
+    - $R -e "library(DoubletFinder)"
+
+about:
+  home: https://github.com/chris-mcginnis-ucsf/DoubletFinder
+  license: unknown
+  summary: R package for detecting doublets in single-cell RNA sequencing data
+extra:
+  additional-platforms:
+    - linux-aarch64
```

Comment:
`additional-platforms` is not needed for `noarch` recipes

Reply:
Ah, good to know, thanks